File: support/src/main/java/io/wax911/support/base/animation/AnimationDuration.java
package io.wax911.support.base.animation;

import androidx.annotation.IntDef;
import kotlin.annotation.AnnotationRetention;
import kotlin.annotation.Retention;

@Retention(AnnotationRetention.SOURCE)
@IntDef({AnimationDuration.SHORT, AnimationDuration.MEDIUM, AnimationDuration.LONG})
public @interface AnimationDuration {
    int SHORT = 250;
    int MEDIUM = 500;
    int LONG = 750;
}

File: support/src/main/java/io/wax911/support/base/animation/AnimationDuration.kt
package io.wax911.support.base.animation

import androidx.annotation.IntDef

@Retention(AnnotationRetention.SOURCE)
@IntDef(AnimationDuration.SHORT, AnimationDuration.MEDIUM, AnimationDuration.LONG)
annotation class AnimationDuration {
    companion object {
        const val SHORT = 250
        const val MEDIUM = 500
        const val LONG = 750
    }
}
